src/main/java/com/stripe/model/AccountCreateParams.java

```diff
@@ -0,0 +1,182 @@
+package com.stripe.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class AccountCreateParams {
+
+  private String type;
+  private LegalEntity legalEntity;
+
+  /**
+   * Account create params internal constructor.
+   */
+  AccountCreateParams(String type, LegalEntity legalEntity) {
+    this.type = type;
+    this.legalEntity = legalEntity;
+  }
+
+  public static Builder builder() {
+    return new Builder();
+  }
+
+  public static class Builder {
+    private String type;
+    private LegalEntityBuilder legalEntityBuilder = new LegalEntityBuilder(this);
+
+    Builder() {}
+
+    public Builder setType(String type) {
+      this.type = type;
+      return this;
+    }
+
+    public LegalEntityBuilder beginLegalEntity() {
+      return legalEntityBuilder;
+    }
+
+    public AccountCreateParams build() {
+      return new AccountCreateParams(type, legalEntityBuilder.build());
+    }
+  }
+
+  /**
+   * Inner classes and their builders.
+   */
+  @Getter
+  @Setter
+  static class AdditionalOwner {
```

Comment:
Each internal class has package visibility and hidden from the users.

Reply:
It's not clear to me what this buys us when the builder for the object itself is public